core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2636,13 +2636,6 @@ final class Stream[+F[_], +O] private (private val free: FreeC[Nothing, O, Unit]
   )(f: (Stream[F, O], Stream[F2, O2]) => Stream[F2, O3]): Stream[F2, O3] =
     f(this, s2)
 
-  /**
-    * Applies the given sink to this stream.
-    */
-  @deprecated("Use .through instead", "1.0.2")
-  private[fs2] def to[F2[x] >: F[x]](f: Stream[F, O] => Stream[F2, Unit]): Stream[F2, Unit] =
```

Comment:
This is the reason for the mima exclusion. Not related to #1709 but noticed we can remove this safely now.